src/libpgmoneta/wal.c

```diff
@@ -420,6 +424,11 @@ pgmoneta_wal(int srv, char** argv)
       pgmoneta_consume_data_row_messages(ssl, socket, buffer, &end_of_timeline_response);
       timeline = atoi(pgmoneta_query_response_get_data(end_of_timeline_response, 0));
       xlogpos = pgmoneta_query_response_get_data(end_of_timeline_response, 1);
+
+      // Assign the current xlogpos to LSN
+      memset(config->servers[srv].pgmoneta_current_wal_lsn, 0, MISC_LENGTH);
+      snprintf(config->servers[srv].pgmoneta_current_wal_lsn, MISC_LENGTH, "%s", xlogpos);
+
```

Comment:
@Jubilee101 Would you like me to move this block after the xlogpos validation too?

Reply:
Yup, that would be great